### ISSUE TITLE
Dark mode

### ISSUE THREAD

**Issue by bergie** [SignalK/instrumentpanel#109]
Many modern devices can support a dark mode with the [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) media query. Adding dark styling to the instrument panel utilizing this would allow automatic switching to a more night vision friendly coloring at night time.

Quick-and-dirty option would be:

```css
@media (prefers-color-scheme: dark) {
  body {
    filter: invert(100%);
    background-color: #000;
  }
}
```

This would look like:
<img width="338" alt="Screenshot 2019-09-16 at 14 54 52" src="https://user-images.githubusercontent.com/3346/64959701-fabc1b00-d891-11e9-8d46-ff751a89673f.png">

<img width="513" alt="Screenshot 2019-09-16 at 14 56 05" src="https://user-images.githubusercontent.com/3346/64959771-263f0580-d892-11e9-9afb-2102f2baa235.png">

Of course one could design a nicer color scheme, too.

Relates to #30

**Comment by cmotelet** [comment]
I am glad to know that you use instrumentpanel also at night.
I had already thought about this option without knowing if it would be of interest to users.
I'm putting this feature at the top of the list. It will be added in the next version.

**Comment by bergie** [comment]
BTW, the way Freeboard-SK did it is pretty nice: https://github.com/SignalK/freeboard-sk/blob/6ee94482d5982300aa34c4c1975a8734da4d24d0/CHANGELOG.md

- **Added:** Alternate color scheme (`Dark mode`) configuration setting with options to use either the: 
    1) Device OS setting via the `prefers-color-scheme` css media attribute
    2) Signal K `environment.mode` value.

**Comment by cmotelet** [comment]
Thanks Bergie, I had seen and tested it as soon as it went online.
The dark mode for IP is in progress...
I work on css and cleaning hard coded colors.
I will most likely use the same options as `freeboard-sk` with maybe an extra button for the manual switching.
https://github.com/cmotelet/instrumentpanel/tree/darkMode-dev
![image](https://user-images.githubusercontent.com/31212553/67040339-90fa8f80-f123-11e9-816a-f6f22a2fb4e6.png)
![image (1)](https://user-images.githubusercontent.com/31212553/67040370-a96aaa00-f123-11e9-89ac-c8080152d3ee.png)

**Comment by cmotelet** [comment]
Closed by #118
